package.json

```diff
@@ -43,7 +43,7 @@
     "@types/chai-enzyme": "^0.6.2",
     "@types/chart.js": "^2.4.6",
     "@types/classnames": "0.0.32",
-    "@types/d3-scale": "^2.0.0",
+    "@types/d3-scale": "^2.0.1",
```

Comment:
@adamabeshouse when we do a fresh node install, these dependencies will get updated automatically. This is ok?

Reply:
i think so... as long as tests pass i guess its fine